.vscode/settings.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "eslint.experimental.useFlatConfig": true
```

Comment:
there is a pre-release version of `vscode-eslint` which supports flat config - https://github.com/microsoft/vscode-eslint?tab=readme-ov-file#version-305---pre-release. it can be enabled in `vscode-eslint` extension settings in vs code